Frontend/ui-customer/src/Screens/OnBoardingFlow/SuccessScreen/View.purs

```diff
@@ -108,7 +108,7 @@ lottieLoaderView state push =
     [ id (EHC.getNewIDWithTag "SuccessLottieView")
     , afterRender
         ( \action -> do
-            _ <- pure $ startLottieProcess "success_lottie" (EHC.getNewIDWithTag "SuccessLottieView") true 1.0 "default"
+            _ <- pure $ startLottieProcess animationConfig{ rawJson = "success_lottie", lottieId = (EHC.getNewIDWithTag "SuccessLottieView") }
```

Comment:
@ashishsingh101 can you override the value of speed also here. As we need `1.0` and the default speed is `0.6`.

Reply:
Check at other places as well. Wherever it is applicable.